reconcile/terraform_users.py

```diff
@@ -210,7 +212,30 @@ def run(
 
 
 def early_exit_desired_state(*args, **kwargs) -> dict[str, Any]:
+    def add_account_identity(acc):
```

Comment:
nit: It might be good to have a comment here why this is necessary. It made sense to my now after you explained it to me :)

```
"""
Finding diffs in deeply nested structures is time/resource consuming.
Having unique known properties makes it easier to match the same object in different states.
I.e., we add a known unique property here to speed-up diff search in a later step.
"""
```

Reply:
fixed